### ISSUE TITLE
Replace audio file names in official mods rules with references to notifications/voices.yaml entries

### ISSUE THREAD

**Issue by penev92** [OpenRA/OpenRA#10663]
A split from the generic D2k cleanup ticket:
>There are still actor definitions that contain hardcoded (in the YAML, but still) sound file names like `HI_PREP.AUD` and `STEALTH1.WAV` instead of properly defined sounds/voices/notifications in the proper YAML file. This is also true for other mods and I think it would be nice to clean those up, because they have been known to cause problems.

**Comment by reaperrr** [comment]
We need to be a bit careful about stuff like stealth sounds, because these need to be WPos-specific.

**Comment by pchote** [comment]
I believe this is about moving them into Notification/Sounds (that can then still be played at a specific position).

**Comment by reaperrr** [comment]
Ok, I wasn't sure whether our notification system supports that already.

**Comment by GraionDilach** [comment]
What is the gain? The loss is to work with another file all the time and being a lot less modder friendly.

**Comment by penev92** [comment]
`¯\_(ツ)_/¯ ` Looks cleaner to me, and you only define every file once that way.
This was more of a suggestion than a "we *have* to do it".

**Comment by abcdefg30** [comment]
Tbh, I think that's a good question. It really seems to add just extra work. The only benefits I can think of are translation support for speech notifications/voices (why do we need sounds then?) and that mappers have an overview which notifications they can use (https://github.com/OpenRA/OpenRA/pull/7164#issuecomment-68636229).

**Comment by pchote** [comment]
Consistency is an important point.  It also exposes them to the linter, and will make for an easier transition to the improved sound sequence setup (when that eventually happens).

**Comment by Apollo-X** [comment]
I presume a sound definition yaml is in the plans then? Being able to specify volume and other key characteristics like range of said audio would go long way and potential randomization of sound/pitch etc used.

**Comment by pchote** [comment]
@Apollo-X: yup.  We will have a setup similar to our artwork sequences where you have a named sound type, with properties for things like faction-specific naming conventions, sound file randomization, volume, etc.

**Comment by GraionDilach** [comment]
If at least some basic options can arrive along with the refactor then it is a gain.
